react-context/client/src/pages/AuthPage.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import React, { useContext, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { FaBolt } from 'react-icons/fa';
+import AuthForm from '../components/AuthForm';
+import AppContext from '../components/AppContext';
+
+export default function AuthPage({ action }) {
+  const navigate = useNavigate();
+  /* TODO: Grab `user` and `handleSignIn` from `AppContext` */
+  const { user } = useContext(AppContext);
+  const { handleSignIn } = useContext(AppContext);
```

Comment:
You can destructure both in one statement:
`const { user, handleSignIn } = useContext(AppContext);`